### ISSUE TITLE
FillVerticalSpace isn't evenly distributing vspace?

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1655]
[functions-tables.pdf](https://github.com/bootstrapworld/curriculum/files/13429154/functions-tables.pdf)

**Comment by schanzer** [comment]
@flannery-denny can you add some details to this issue? What are you expecting the page to look like?

**Comment by schanzer** [comment]
@flannery-denny I've found what I believe is the culprit (wrong CSS rule for distributing grid height). Are you see this issue in a lot of other pages, or just a handful?

**Comment by flannery-denny** [comment]
I think just a handful. But I can do a full workbook scan and make a list if you want.

**Comment by schanzer** [comment]
@flannery-denny I spent most of today trying to come up with a heuristic that would Do the Right Thing(tm) for all of our tables. Ultimately I failed -- our use-cases are just too varied to use auto-detection, so it's going to fall on authors to say what they need.

I've just pushed a new style rule, and an example of how to use it in the example file you shared. If this works for you, it's easy to add to any other relevant pages.

It's opt-in, which means that no other pages need to be checked.
